### PR TITLE
Chore/proposals voting

### PR DESCRIPTION
**Pull request by nguernse**
Main goal for this PR was to update the `proposals.index` to return the logged in users vote.

Other additions:
- Tests for proposals endpoint
- Added seed scripts for test environment and for local development
- Updated `README` to reflect updates

@randomontherun

### CODE REVIEW

File: tests/proposals.test.ts [hunk 1]
```diff
@@ -0,0 +1,134 @@
+import { it, describe, expect } from "vitest";
+import { TEST_SERVER_URL } from "./global.setup";
+import { resetDatabase } from "../database";
+import { Proposal } from "../types/proposal";
+import { seedDatabase } from "./helpers/seedDatabase";
+import { TEST_USER } from "../helpers";
+import { beforeEach } from "node:test";
```
Comment (randomontherun) [inline tests/proposals.test.ts:7]: I think vitest has a beforeEach function

**Review by randomontherun — CHANGES_REQUESTED** [review]
This all looks great. I'm only requesting you remove the comments. They were helpful, but shouldn't be in production code. No need to remove the labels, they look good.

**Review by randomontherun — APPROVED** [review]
Looks great!

File: tests/proposals.test.ts (hunk 1 above)
Comment (zenlex) [reply tests/proposals.test.ts]: it does

**Review by zenlex — APPROVED** [review]

**Pull request merged**